docs/orchestrate/saucectl-configuration.md

```diff
@@ -375,6 +356,72 @@ artifacts:
 
 ---
 
+## `reporters`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Configures additional reporting capabilities provided by `saucectl`.
+
+```yaml
+reporters:
+  json:
+    enabled: true
+    filename: saucectl-report.json
+```
+---
+
+### `json`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The JSON reporter creates a single report of all executed saucectl suites.
+
+```yaml
+reporters:
+  json:
+    enabled: true
+    filename: saucectl-report.json
+    webhookURL: https://my-webhook-url
+```
+
+---
+
+#### `enabled`
+
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Toggles the reporter on/off.
+
+```yaml
+    enabled: true
```

Comment:
We previously discussed that future entries like these should contain the parent entity as well, otherwise it's hard to read and copy-paste samples:
```suggestion
reporters:
  json:
    enabled: true
```

Reply:
OK. Then I think the rest of the docs should be fixed as well? There are a lot similar config fields like this.

Reply:
That'd be great!